src/Uno.Toolkit.UI/Behaviors/CommandExtensions.cs

```diff
@@ -137,16 +150,54 @@ internal static bool TryInvokeCommand(DependencyObject owner, object? parameter)
 
 			return false;
 		}
+		internal static object? TryGetItemCommandParameter(DependencyObject? container)
+		{
+			if (container is ListViewItem)
+			{
+				// fixme: it doesn't work here, because we came from ListView::ItemClick,
+				// and when that happens the SelectedIndex not yet set.
```

Comment:
to convert to an issue

Reply:
@Xiaoy312 if you already converted it to an issue, maybe you can add the related link in the comment here

Reply:
[@Xiaoy312](https://github.com/Xiaoy312) mentioned this pull request [yesterday](https://github.com/unoplatform/uno.toolkit.ui/pull/1170#ref-issue-2379020100)
https://github.com/unoplatform/uno.toolkit.ui/issues/1171